exoscale/api/compute.py

```diff
@@ -1035,6 +1035,21 @@ def delete(self):
         self.date = None
         self.size = None
 
+    def export(self):
+        """
+        Exports the storage volume snapshot.
+
+        Returns:
+            str: the URL of the exported snapshot file
+        """
+
+        try:
+            res = self.compute.cs.exportSnapshot(id=self.id)
+        except CloudStackApiException as e:
+            raise APIException(e.error["errortext"], e.error)
+
+        return res["presignedurl"]
```

Comment:
```suggestion
        return res
```

It seems there won't always just be a pre-signed URL in the response, but possibly some more metadata.

Reply:
Changed in bf307f2